grapher/captionedChart/CaptionedChart.tsx

```diff
@@ -252,7 +252,10 @@ export class CaptionedChart extends React.Component<CaptionedChartProps> {
         if (manager.showZoomToggle)
             controls.push(<ZoomToggle key="ZoomToggle" manager={manager} />)
 
-        if (manager.showFacets) {
+        if (
+            manager.showFacets &&
+            manager.availableFacetStrategies!.length > 1
```

Comment:
Miraculously, this one was easy. The other parameters could not be made required without affecting a huge number of places, because of the interface inheritance pattern.

Reply:
Sorry, removed! I actually wish a redundant ! was a linting error.